src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <value>Learn more about Individual Donations in Trailhead</value>
+    </labels>
+     <labels>
+    <fullName>gseuChecklistItemDonationIndividualPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Individual in GS for End User</shortDescription>
+        <value>Enter Individual Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Individual in GS for End User</shortDescription>
+        <value>Enter a Donation from an Individual</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Institutional and corporate giving are an important source of nonprofit support. Track organizational giving by creating an Opportunity from an Account record.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Learn more about Organization Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Enter Organization Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Enter a Donation from an Organization</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Track Donation in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Description of Track Donation checklist item in Get Started for End Users.